src/VscPreviewExtension/DfmParse/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.dll
+*.exe
+*.config
```

Comment:
But I want to upload the empty 'DfmParse' folder for that some .exe and .dll  files will be copied to this folder from ‘src/Microsoft.DocAsCode.Dfm.VscPreview/' . Is there anothor way to achieve this. 

---
In reply to: [74867922](https://github.com/dotnet/docfx/pull/609#discussion_r74867922) [](ancestors = 74867922)